compass/ocean/__init__.py

```diff
@@ -26,3 +27,4 @@ def __init__(self):
         self.add_test_group(IceShelf2d(mpas_core=self))
         self.add_test_group(IsomipPlus(mpas_core=self))
         self.add_test_group(Ziso(mpas_core=self))
+        self.add_test_group(SphereTransport(mpas_core=self))
```

Comment:
```suggestion
        self.add_test_group(SphereTransport(mpas_core=self))
        self.add_test_group(Ziso(mpas_core=self))
```
One more super nit-picky thing.  I'd like to keep the test groups alphabetical.